<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LineCameraClass</class>
 <widget class="QMainWindow" name="LineCameraClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1330</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LineCamera</string>
  </property>
  <property name="windowIcon">
   <iconset resource="LineCamera.qrc">
    <normaloff>:/LineCamera/Resources/公司LOG.jpg</normaloff>:/LineCamera/Resources/公司LOG.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>1311</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,0,0,0,0,0,0,0,0">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>   模式</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_10">
         <property name="text">
          <string>过</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;上限x：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_4"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;下限y:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_3"/>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;测量值：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2"/>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>device</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_4">
         <property name="text">
          <string>show</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_3">
         <property name="text">
          <string>停止检测</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_5">
         <property name="text">
          <string>link</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>暂停</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_6">
         <property name="text">
          <string>算法设定</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_7">
         <property name="text">
          <string>运动设定</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_8">
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_9">
         <property name="text">
          <string>设备检测</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>启动检测</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1311</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>16</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>楷体</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;颗粒物杂质光学检查机&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>WELCOME TO USE</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;上海迈迅威视觉科技有限公司&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>1301</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1330</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>pushButton_8</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton_7</tabstop>
  <tabstop>pushButton_9</tabstop>
 </tabstops>
 <resources>
  <include location="LineCamera.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>LineCameraClass</receiver>
   <slot>showimage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>LineCameraClass</receiver>
   <slot>btnopen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>629</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>LineCameraClass</receiver>
   <slot>btnstop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>880</x>
     <y>629</y>
    </hint>
    <hint type="destinationlabel">
     <x>377</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>LineCameraClass</receiver>
   <slot>btnclose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1318</x>
     <y>629</y>
    </hint>
    <hint type="destinationlabel">
     <x>495</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>LineCameraClass</receiver>
   <slot>linkdevice()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>880</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>LineCameraClass</receiver>
   <slot>setAlgorithm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>724</y>
    </hint>
    <hint type="destinationlabel">
     <x>82</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>LineCameraClass</receiver>
   <slot>setconsole()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>880</x>
     <y>724</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>LineCameraClass</receiver>
   <slot>getproduct()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1318</x>
     <y>666</y>
    </hint>
    <hint type="destinationlabel">
     <x>553</x>
     <y>459</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_9</sender>
   <signal>clicked()</signal>
   <receiver>LineCameraClass</receiver>
   <slot>test()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1318</x>
     <y>715</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_4</sender>
   <signal>editingFinished()</signal>
   <receiver>LineCameraClass</receiver>
   <slot>geta()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>549</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_3</sender>
   <signal>editingFinished()</signal>
   <receiver>LineCameraClass</receiver>
   <slot>getb()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>771</x>
     <y>549</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>LineCameraClass</receiver>
   <slot>getmodel(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>555</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_10</sender>
   <signal>clicked()</signal>
   <receiver>LineCameraClass</receiver>
   <slot>forceGo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>563</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>557</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showimage()</slot>
  <slot>btnopen()</slot>
  <slot>btnclose()</slot>
  <slot>btnstop()</slot>
  <slot>linkdevice()</slot>
  <slot>setAlgorithm()</slot>
  <slot>setconsole()</slot>
  <slot>getproduct()</slot>
  <slot>test()</slot>
  <slot>geta()</slot>
  <slot>getb()</slot>
  <slot>getmodel(int)</slot>
  <slot>forceGo()</slot>
 </slots>
</ui>
